migration/migrator/migrations/master/20221206153408_add_user_preferred_name_order.py

```diff
@@ -0,0 +1,27 @@
+"""Migration for the Submitty master database."""
+
+
+def up(config, database):
+    """
+    Run up migration.
+
+    :param config: Object holding configuration details about Submitty
+    :type config: migrator.config.Config
+    :param database: Object for interacting with given database for environment
+    :type database: migrator.db.Database
+    """
+    sql = "ALTER TABLE ONLY users ADD COLUMN IF NOT EXISTS user_preferred_name_order SMALLINT NOT NULL DEFAULT 0;"
+    database.execute(sql)
+
+
+def down(config, database):
+    """
+    Run down migration (rollback).
+
+    :param config: Object holding configuration details about Submitty
+    :type config: migrator.config.Config
+    :param database: Object for interacting with given database for environment
+    :type database: migrator.db.Database
+    """
+    sql = "ALTER TABLE ONLY users DROP COLUMN IF EXISTS user_preferred_name_order RESTRICT;"
+    database.execute(sql)
```

Comment:
In Submitty, we usually prefer that our down migrations don't delete data unless absolutely necessary for functionality. This this case we'd probably want the down migration to just be `pass`.